CMSgov/design-system
# [WNMGDS-1330] countdown with storage

## Summary

Re-implementing the countdown functionality with local storage instead of settimeouts. This also includes unit tests and updates to the Dialog component to make the 'Close' optional

## How to test

Run `yarn storybook` to see the component in action. Run `yarn test` to run unit tests
